muttdown/main.py

```diff
@@ -27,24 +29,28 @@ def convert_one(part, config):
         text = part.get_payload(decode=True)
         if not isinstance(text, six.text_type):
             # no, I don't know why decode=True sometimes fails to decode.
+            # it is because its decoding from different formats. first base64 then unicode ---guygma
             text = text.decode('utf-8')
         if not text.startswith('!m'):
             return None
         text = re.sub('\s*!m\s*', '', text, re.M)
+        f = tempfile.NamedTemporaryFile(suffix='_panmail.css')
```

Comment:
temporary files should always be used as context managers so that they get cleaned up on pypy

Reply:
This bothered me as well -- to be sure. I did initially implement it that way. The reason I changed was to try and change less of the underlying code structure. I think handling the signature separately is where I ran into an awkward situation so  found my way around it by just manually closing the file later (which also deletes it). Can do better on that though.